test/src/test/java/org/corfudb/infrastructure/SequencerServerTest.java

```diff
@@ -204,7 +204,7 @@ public void SequencerWillResetTails() throws Exception {
         // Modifying the readyStateEpoch to simulate sequencer reset.
         server.setReadyStateEpoch(-1L);
         sendMessage(new CorfuPayloadMsg<>(CorfuMsgType.BOOTSTRAP_SEQUENCER,
-                new SequencerTailsRecoveryMsg(globalTail + 2, tailMap, 0L)));
+                new SequencerTailsRecoveryMsg(globalTail + 2, tailMap, 0L, false)));
```

Comment:
The usage can be a bit confusing. The above example means a bootstrap of NOT_READY sequencer while if we pass true, it will be actually an update of an existing sequencer and not a bootstrap in spite of the message type being BOOTSTRAP_SEQUENCER.

Reply:
I have updated the name of the variable to be more intuitive.
`bootstrapWithoutTailsUpdate` - If set to true it bootstraps without the tails update payload.
Else it just updates the epoch.